cluster/manifests/skipper/deployment.yaml

```diff
@@ -95,7 +95,7 @@ spec:
           - "-response-header-timeout-backend={{ .ConfigItems.skipper_response_header_timeout_backend }}"
           - "-timeout-backend={{ .ConfigItems.skipper_timeout_backend }}"
           - "-tls-timeout-backend={{ .ConfigItems.skipper_tls_timeout_backend }}"
-          - "-access-log-disabled"
+          - "-default-filters-prepend='enableAccessLog(4,5)'"
```

Comment:
I think you need to remove the `''` otherwise the flag is not parsed:

```
invalid value "'enableAccessLog(4,5)'" for flag -default-filters-prepend: failed to parse default filters: parse failed after token ->, position 5: syntax error
```

Reply:
thanks, trying